src/api/Annotations.js

```diff
@@ -24,9 +25,26 @@ import type {
 } from '../common/types/annotations';
 import type { BoxItemPermission } from '../common/types/core';
 import type { ElementsXhrError } from '../common/types/api';
-import type { FeedItemStatus } from '../common/types/feed';
+import type { Comment, FeedItemStatus, ThreadedComments as ThreadedCommentsType } from '../common/types/feed';
 
 export default class Annotations extends MarkerBasedApi {
+    /**
+     * Formats annotation replies' comment data for use in components.
+     *
+     * @param {Annotation} annotation - An individual annotation entry from the API
+     * @return {Annotation} Updated annotation
+     */
+    format(annotation: Annotation): Annotation {
```

Comment:
Should we rename this to `formatReplies`?

Reply:
Makes sense. Renamed.